frontend/model/state.js

```diff
@@ -140,6 +140,11 @@ const mutations = {
     sbp('okTurtles.events/emit', EVENTS.CONTRACTS_MODIFIED, state.contracts)
   },
   setContractHEAD (state, { contractID, HEAD }) {
+    const contract = state.contracts[contractID]
+    if (!contract) {
+      console.error(`This contract ${contractID} doesnt exist anymore. Probably you left the group just now.`)
+      return
```

Comment:
This console.error is always triggered when we are the members being removed. I didn't find a way to avoid it. Need your help @taoeffect